Makefile

```diff
@@ -1,4 +1,44 @@
+.PHONY: clean distclean
```

Comment:
ROOT has moved to cmake. Probably we should switch to cmake, and maybe you want to look at ROOT_STANDARD_PACKAGE macro in RootMacros.cmake file.